docs/src/content/hardhat-network/docs/overview/index.md

```diff
@@ -15,7 +15,7 @@ It runs as either an in-process or stand-alone daemon, servicing JSON-RPC and We
 
 By default, it mines a block with each transaction that it receives, in order and with no delay.
 
-It's backed by the `@ethereumjs/vm` EVM implementation, the same one used by ganache and Remix.
+It's backed by [EDR](https://github.com/NomicFoundation/edr), a custom EVM implementation written in Rust and targeting performance.
```

Comment:
Should we underline the fact that this custom implementation is made by us, the Hardhat team, or is it already implicit?